EIPS/eip-draft-whisper.md

```diff
@@ -0,0 +1,181 @@
+## Preamble
+
+    EIP: draft
+    Title: Whisper Specification
+    Author: Vlad Gluhovsky <gluk256@gmail.com>
+    Type: Informational
+    Status: Draft
+    Created: 2017-05-05
+
+## Abstract
+
+This draft EIP describes the format of Whisper messages within the ÐΞVp2p Wire Protocol.
+This EIP should substitute the [existing specification](https://github.com/ethereum/wiki/wiki/Whisper-Wire-Protocol).
+More detailed documentation on Whisper could be found [here](https://github.com/ethereum/go-ethereum/wiki/Whisper).
+
+## Motivation
+
+It is necessary to specify the standard for Whipser messages in order to ensure forward compatibility of different Whisper clients.
+
+## Specification
+
+All Whisper messages sent as ÐΞVp2p Wire Protocol packets should be RLP-encoded arrays of data containing two objects: integer packet code followed by another object (whose type depends on the packet code). 
+
+If Whisper node does not support a particular packet code, it should just ignore the packet without generating any error.
+
+### Packet Codes
+
+The message codes reserved for Whisper protocol: 0 - 127.
```

Comment:
What's the rationale behind reserving 128 codes?

Reply:
we need to decide upon some arbitrary number. 
128 is as good as any other.

Reply:
@gluk256 this choice leads to integer overflow issues in certain circumstances (see https://github.com/paritytech/parity/issues/7567), so it'd be really great to figure out the motivation for such a wide space and if it is possible to reduce it.